build/Dockerfile

```diff
@@ -20,13 +20,14 @@ ARG LINUX_KERNEL_VERSION
 ARG ROM_VERSION
 
 # Build directories.
-ARG MACHINE_SNAPSHOT_BUILD_PATH=/build/machine-snapshot
+ARG SNAPSHOT_BUILD_PATH=/build/snapshot
 ARG DEVNET_BUILD_PATH=/build/devnet
 ARG RUST_BUILD_PATH=/build/rollups-node/rust
 ARG GO_BUILD_PATH=/build/rollups-node/go
 
 # Runtime dir for the cartesi-machine snapshot.
-ARG MACHINE_SNAPSHOT_RUNTIME_PATH=/usr/share/cartesi/snapshots
+ARG SNAPSHOT_RUNTIME_PATH=/usr/share/cartesi/snapshots
+ARG SNAPSHOT_FILENAME=test-application
```

Comment:
`_FILENAME` is weird since it's a directory.

If the `SNAPSHOT_RUNTIME_PATH` is gonna hold only a single snapshot, maybe renaming it to `/usr/share/cartesi/snapshot` and don't use `SNAPTHOT_FILENAME`.